mne/viz/_3d.py

```diff
@@ -1065,15 +1171,28 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
     Returns
     -------
     brain : Brain
-        A instance of surfer.viz.Brain from PySurfer.
-    """
-    import surfer
-    from surfer import Brain, TimeViewer
-    import mayavi
-
+        A instance of surfer.viz.Brain from PySurfer or matplotlib figure.
+    """  # noqa: E501
     # import here to avoid circular import problem
     from ..source_estimate import SourceEstimate
-
+    import surfer
+    from surfer import Brain, TimeViewer
```

Comment:
I suspect that these will not work with `mayavi` not installed, so you could put them after the `try` below